### PR TITLE
MAGECLOUD-5081: Stabilize Travis CI tests

### PR DESCRIPTION
**Pull request by shiftedreality**
### Fixed Issues (if relevant)

1. https://magento2.atlassian.net/browse/MAGECLOUD-5081

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by oshmyheliuk — APPROVED** [review]

**Review by BaDos — APPROVED** [review]

**Review by bbatsche — APPROVED** [review]

**Comment by shiftedreality** [comment]
No QA needed as no functional changes

File: tests/functional/Codeception/Docker.php [hunk 1]
```diff
@@ -339,10 +339,14 @@ public function addEceComposerRepo(): bool
                 $commands[] = $this->taskComposerRequire('composer')
                     ->dependency($extra['name'], $config['require'][$extra['name']])
                     ->noInteraction()
+                    ->option('--no-update')
                     ->getCommand();
             }
         }
 
+        $commands[] = $this->taskComposerUpdate('composer')
```
Comment (NadiyaS) [inline tests/functional/Codeception/Docker.php:347]: This method (addEceComposerRepo () ) is always runs together with cloneTemplate() where taskComposerUpdate() also runs. Probably need to combine them in some way, e.g. remove taskComposerUpdate() from both methods and create one method which will call cloneTemplate(), addEceComposerRepo() and taskComposerUpdate()

Comment (shiftedreality) [reply tests/functional/Codeception/Docker.php:347]: This should be safe, as Composer won't run subsequent `composer update`. In any case, I will add a task for this.

Comment (BaDos) [reply tests/functional/Codeception/Docker.php:347]: @NadiyaS we can leave it as it is. After refactoring FT these functionalities will be removed.

Comment (NadiyaS) [reply tests/functional/Codeception/Docker.php:347]: ok then

**Review by NadiyaS — APPROVED** [review]

**Pull request merged**